src/main/resources/templates/c_plus_plus/test/tests/TestClangFormat.py

```
@@ -0,0 +1,56 @@
+
+    projectRoot: str
+    filePath: str
+
+    def __init__(
+        self,
+        projectRoot: str,
+        filePath: str,
+        requirements: Optional[List[str]] = None,
+        name: str = "TestClangFormat",
+    ) -> None:
+        super(TestClangFormat, self).__init__(
```

Ruff

Use `super()` instead of `super(__class__, self)`

Remove `__super__` parameters

(UP008)